src/routes/auth.js

```diff
@@ -1,15 +1,34 @@
 const User = require('../models/user');
 const userRouter = require('express').Router();
+
+const passport = require('passport');
+
+const FacebookStrategy = require('passport-facebook').Strategy;
+const findOrCreate = require('mongoose-findorcreate');
+
+const {createVerificationLink} = require('../utils/EmailVerification');
+
+
+
+// guestRouter.get('/active', auth, (req, res) => {
+//   res.status(200).json({
+//     _id: req.user.id,
+//     isAdmin: req.user.isEmailVerified,
+//     isAuth: true,
+//     email: req.user.email,
+//     username: req.user.username,
+//   });
+// });
+
 const {
   registerValidation,
   loginValidation,
   forgotValidation,
   resetPasswordValidation
 } = require('../utils/validation/joiValidation');
 const { auth } = require('../utils/middleware');
-const { createVerificationLink } = require('../utils/EmailVerification');
+
 const { userForgotPassword, userResetPassword } = require('../controllers/auth');
-const SessionMgt = require('../services/SessionManagement');
```

Comment:
Is there a reason why u removed the session management service?

Reply:
There were 2 instances I only removed one to resolve a conflict